proposals/configuration-level-resources.md

```diff
@@ -67,13 +79,380 @@ spec:
 The Kubernetes scheduler will schedule the Pod to the Node since the resources is available.
 The kubelet on that Node will make an `allocate` call to the `akri.sh/akri-onvif` device plugin in the Akri Agent.
 The Agent will then reserve slots in the associated Instance device plugins (`akri.sh/akri-onvif-8120fe` and `akri.sh/akri-onvif-a19705`).
-When selecting the slots from Instances, the Agent will preference unique Instances; for example, in the above Pod, the Agent will make sure 1 slot of each `akri-onvif-8120fe` and `akri-onvif-a19705` is reserved for each Pod.
+How the Agent select slots to reserve is based on the value of `Configuration.uniqueDevices`. 
+
+ When `Configuration.uniqueDevices` is true, the Agent will select max 1 slot for each Instance. For example, in the above Pod, the Agent will make sure 1 slot of each `akri-onvif-8120fe` and `akri-onvif-a19705` is reserved for each Pod.
+
+ When `Configuration.uniqueDevices` is false, the Agent will select slots based on deviceUsage availability.  In the same example, if both deviceUsage slots of `akri-onvif-8120fe` are reserved by other nodes, the Agent will selecting both slots from `akri-onvif-a19705`, both maps to the same instance.
```

Comment:
I don't like this approach... and maybe this is argument for `uniqueDevices` to be named differently. I still think, even if using deviceUsage or capacity availability, pods should get unique instances. If both slots of `akri-onvif-8120fe` are taken, then the pod should fail to be scheduled. This is where i get a little lost of the reason why `uniqueDevices=true` is a useful scenario